ouroboros-network/src/Ouroboros/Network/NodeToClient.hs

```diff
@@ -587,6 +587,7 @@ networkErrorPolicies = ErrorPolicies
                       MuxDecodeError          -> Just ourBug
                       MuxIngressQueueOverRun  -> Just ourBug
                       MuxInitiatorOnly        -> Just ourBug
+                      MuxShutdown             -> Just ourBug
```

Comment:
Is it? A mux shutdown can be due to protocol errors.

Reply:
That is correct, but that will not result in a "MuxShutdown" exception being thrown.